app/templates/components/profile-nav.hbs

```diff
@@ -161,7 +161,7 @@
               </LinkTo>
             </li>
           {{/if}}
-          {{#if this.showSubscriptionTab}}
+          {{#if this.showPlanUsageTab}}
```

Comment:
In order to hide the Plan Usage tab for Hybrid plans, this check should also be applied on [line#129](https://github.com/travis-ci/travis-web/blob/0c0dd341003e444c0eca05b9809ce85e8cfcf079/app/templates/components/profile-nav.hbs#L129)

Reply:
Missed that, sorry